test/blackbox-tests/test-cases/macro-expand-error.t/run.t

```diff
@@ -1,9 +1,24 @@
 we're getting an acceptable error message when adding a macro form in an
 inappropariate place:
 
+  $ echo '(lang dune 1.0)' > dune-project
+  $ echo '(copy_files %{dep:x}/*)' > dune
+
   $ dune build
-  File "dune", line 1, characters 12-21:
-  1 | (copy_files %{read:x}/*)
-                  ^^^^^^^^^
-  Error: %{read:..} isn't allowed in this position.
+  File "dune", line 1, characters 12-20:
+  1 | (copy_files %{dep:x}/*)
+                  ^^^^^^^^
+  Error: %{dep:..} isn't allowed in this position.
   [1]
+
+And the same if a macro creates a cycle:
+
+  $ echo '(lang dune 3.0)' > dune-project
+  $ echo '(copy_files %{read:x})' > dune
+
+  $ dune build
+  Error: Dependency cycle between:
```

Comment:
Printing "between" in dependency cycle error messages always felt strange to me, because "between" is used when talking about two things, not an arbitrary number of them (like one in this particular case). I'll put up a PR to drop "between" from these messages.

Reply:
Actually, using "between" to indicate a cyclic relationship involving multiple items appears to be fine. It's just the singleton case which is strange. Anyway, this isn't related to this PR.